code/src/components/ProjectCarousel.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Flickity from "react-flickity-component";
+
+const flickityOptions = {
+  initialIndex: 0,
+  pageDots: false,
+  wrapAround: true,
+  freeScroll: true,
+  // autoPlay: 6000,
+};
+
+const ProjectCarousel = (props) => {
```

Comment:
Loving the carousel feature on mobile version! It's easier using it there than trying it in the browser.

Reply:
Completely agree! The to-do list is much more mobile-friendly than it is desktop friendly. I did primarily focus on making it a mobile web-app than I did on desktop-friendliness (except for animations that are only visible on desktop).